src/kw_string.sh

```diff
@@ -193,3 +197,52 @@ function concatenate_with_commas()
 
   printf '%s\n' "$*"
 }
+
+# This function validates the encryption. If the passed encryption is not valid
+# this will warn the user and clear the option.
+#
+# @option: The option to determine if it should be an email
+# @value:  The value being passed
+#
+# Return:
+# Returns 0 if valid; 22 if invalid
+function validate_email()
+{
+  local value="$1"
+
+  if [[ ! "$value" =~ ^${email_regex}$ ]]; then
+    complain "Invalid email: $value"
+    return 22 #EINVAL
+  fi
+
+  return 0
+}
+
+# Validates the recipient list given by the user to the options `--to` and
+# `--cc` to make sure the all the recipients are valid.
+#
+# @raw: The list of email recipients to be validated
+#
+# Return:
+# 22 if there are invalid entries; 0 otherwise
```

Comment:
Please, rewrite this documentation. Keep in mind that you are adding this to the kw_string, which is intended to be generic. Also, add an example.